Program.cs

```diff
@@ -722,6 +722,97 @@ public static void Props()
             Console.Clear();
         }
 
+        //User Powerup and prep
+        static void PowerUp()
+        {
+            Console.WriteLine("=========================================");
+            Console.WriteLine("==  Successful survived from the dark  ==");
+            Console.WriteLine("=========================================");
+            temp = Console.ReadLine();
+
+            while (true)
+            {
+                Console.WriteLine("============================================");
+                Console.WriteLine("==            Entering the city           ==");
+                Console.WriteLine("==   1: Booking a room and have a rest    ==");
+                Console.WriteLine("==         2: Go and get HP energy        ==");
+                Console.WriteLine("==   3: Buy tools knifes, rods and fire   ==");
+                Console.WriteLine("============================================");
+                switch (temp)
+                {
+                    case "1":
+                        HP = HP + 40;
```

Comment:
The HP variables here are never initialized. I initialized a variable for the combat branch as playerHP so you can either use that one or I can change mine to this naming convention. Either way is fine with me.